On-premises SOAR end-of-life release note (#3272)
SumoLogic/sumologic-documentation: blog-csoar/2023-11-01-23-application-update.md

---
title: November 1, 2023 - Application Update
hide_table_of_contents: false
keywords:
  - sumo logic
  - cloud soar
 
tags: [cloud soar release notes, application update]
image: https://help.sumologic.com/img/sumo-square.png
---

import useBaseUrl from '@docusaurus/useBaseUrl';

### Sumo Logic On-Premises SOAR Solution End-of-Life

As of **November 15, 2023**, Sumo Logic's on-premises SOAR solution no longer receives updates, and Sumo Logic Engineering no longer develops, repairs, maintains, or tests the software. 

> On **December 1, 2024**, Sumo Logic’s on-premises SOAR solution reaches its end-of-life and is obsolete. Beginning on that date, it no longer receives applicable support entitled by active support contracts or by applicable warranty terms and conditions. 

Reviewer: ```suggestion
Effective **December 1, 2024**, Sumo Logic’s on-premises SOAR solution reaches end-of-life and becomes obsolete. Beginning on that date, it no longer receives applicable support entitled by active support contracts or by applicable warranty terms and conditions. 
```

> To continue to utilize Sumo Logic’s SOAR capabilities, use our SAAS product, [Cloud SOAR](/docs/cloud-soar/). 

Reviewer: ```suggestion
To continue to utilize Sumo Logic’s SOAR capabilities, use our SaaS product, [Cloud SOAR](/docs/cloud-soar/). 
```
